---
title: "Novel variation and de novo mutation rates in population-wide de novo assembled Danish trios"
authors:
- Søren Besenbacher
- Siyang Liu
- José MG Izarzugaza
- Jakob Grove
- Kirstine Belling
- Jette Bork-Jensen
- Shujia Huang
- Thomas D Als
- Shengting Li
- Rachita Yadav
- Arcadio Rubio-García
- Francesco Lescai
- Ditte Demontis
- Junhua Rao
- Weijian Ye
- Thomas Mailund
- Rune M Friborg
- Christian NS Pedersen
- Ruiqi Xu
- Jihua Sun
- Hao Liu
- Ou Wang
- Xiaofang Cheng
- David Flores
- Emil Rydza
- Kristoffer Rapacki
- John Damm Sørensen
- Piotr Chmura
- David Westergaard
- Piotr Dworzynski
- Thorkild IA Sørensen
- Ole Lund
- Torben Hansen
- Xun Xu
- Ning Li
- Lars Bolund
- Oluf Pedersen
- Hans Eiberg
- Anders Krogh
- Anders D Børglum
- Søren Brunak
- Karsten Kristiansen
- Mikkel H Schierup
- Jun Wang
- Ramneek Gupta
- Palle Villesen
- Simon Rasmussen

author_notes:
- "Equal contribution"

date: "2015-01-19"
doi: "10.1038/ncomms6969"

# Schedule page publish date (NOT publication's date).
publishDate: "2015-01-19"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "*Nature communications*"
publication_short: ""

abstract: Building a population-specific catalogue of single nucleotide variants (SNVs), indels and structural variants (SVs) with frequencies, termed a national pan-genome, is critical for further advancing clinical and public health genetics in large cohorts. Here we report a Danish pan-genome obtained from sequencing 10 trios to high depth (50 × ). We report 536k novel SNVs and 283k novel short indels from mapping approaches and develop a population-wide de novo assembly approach to identify 132k novel indels larger than 10 nucleotides with low false discovery rates. We identify a higher proportion of indels and SVs than previous efforts showing the merits of high coverage and de novo assembly approaches. In addition, we use trio information to identify de novo mutations and use a probabilistic method to provide direct estimates of 1.27e−8 and 1.5e−9 per nucleotide per generation for SNVs and indels, respectively.


# Summary. An optional shortened abstract.
summary: ""

tags:
- Interneurons
featured: false

# links:
# - name: ""
#   url: ""
url_pdf: https://www.nature.com/articles/ncomms6969
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ''
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides:
---
