---
title: "Sequencing and de novo assembly of 150 genomes from Denmark as a population reference"
authors:
- Lasse Maretty
- Jacob Malte Jensen
- Bent Petersen
- Jonas Andreas Sibbesen
- Siyang Liu
- Palle Villesen
- Laurits Skov
- Kirstine Belling
- Christian Theil Have
- Jose MG Izarzugaza
- Marie Grosjean
- Jette Bork-Jensen
- Jakob Grove
- Thomas D Als
- Shujia Huang
- Yuqi Chang
- Ruiqi Xu
- Weijian Ye
- Junhua Rao
- Xiaosen Guo
- Jihua Sun
- Hongzhi Cao
- Chen Ye
- Johan Van Beusekom
- Thomas Espeseth
- Esben Flindt
- Rune M Friborg
- Anders E Halager
- Stephanie Le Hellard
- Christina M Hultman
- Francesco Lescai
- Shengting Li
- Ole Lund
- Peter Løngren
- Thomas Mailund
- Maria Luisa Matey-Hernandez
- Ole Mors
- Christian NS Pedersen
- Thomas Sicheritz-Pontén
- Patrick Sullivan
- Ali Syed
- David Westergaard
- Rachita Yadav
- Ning Li
- Xun Xu
- Torben Hansen
- Anders Krogh
- Lars Bolund
- Thorkild IA Sørensen
- Oluf Pedersen
- Ramneek Gupta
- Simon Rasmussen
- Søren Besenbacher
- Anders D Børglum
- Jun Wang
- Hans Eiberg
- Karsten Kristiansen
- Søren Brunak
- Mikkel Heide Schierup

author_notes:
- "Co-first author"
- "Co-first author"
- "Co-first author"
- "Co-first author"
- "Co-first author"
- "Co-first author"
- "Co-first author"
- "Co-first author"
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
- 
- "Correspondence author"
- "Correspondence author"
- "Correspondence author"

date: "2017-08-01"
doi: "10.1038/nature23264"

# Schedule page publish date (NOT publication's date).
publishDate: "2017-08-01"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "*Nature*"
publication_short: ""

abstract: Hundreds of thousands of human genomes are now being sequenced to characterize genetic variation and use this information to augment association mapping studies of complex disorders and other phenotypic traits. Genetic variation is identified mainly by mapping short reads to the reference genome or by performing local assembly. However, these approaches are biased against discovery of structural variants and variation in the more complex parts of the genome. Hence, large-scale de novo assembly is needed. Here we show that it is possible to construct excellent de novo assemblies from high-coverage sequencing with mate-pair libraries extending up to 20 kilobases. We report de novo assemblies of 150 individuals (50 trios) from the GenomeDenmark project. The quality of these assemblies is similar to those obtained using the more expensive long-read technology. We use the assemblies to identify a rich set of structural variants including many novel insertions and demonstrate how this variant catalogue enables further deciphering of known association mapping signals. We leverage the assemblies to provide 100 completely resolved major histocompatibility complex haplotypes and to resolve major parts of the Y chromosome. Our study provides a regional reference genome that we expect will improve the power of future association mapping studies and hence pave the way for precision medicine initiatives, which now are being launched in many countries including Denmark.


# Summary. An optional shortened abstract.
summary: ""

tags:
- Interneurons
featured: false

# links:
# - name: ""
#   url: ""
url_pdf: https://www.nature.com/articles/nature23264
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ''
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides:
---
